### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by tiserett**

### CODE REVIEW

File: src/averageAges.js [hunk 1]
```diff
@@ -55,7 +61,16 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const kids = (onlyWithSon === undefined)
```
Comment (aleksandra-prognimak) [inline src/averageAges.js:64]:
```suggestion
  const kids = !onlyWithSon
```

**Review by aleksandra-prognimak — CHANGES_REQUESTED** [review]
Хорошая работа, но нужно немного улучшить код)

**Review by aleksandra-prognimak — APPROVED** [review]
🔥🔥🔥

File: src/averageAges.js [hunk 2]
```diff
@@ -15,11 +15,12 @@
...
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = !century
+    ? people.filter(person => person.sex === 'm')
+    : people.filter(person => (
+      person.sex === 'm' && Math.ceil(person.died / 100) === century));
+
+  return men.reduce((sum, man) => sum + (man.died - man.born), 0) / men.length;
```
Comment (dellachaise) [inline src/averageAges.js:23]: Here and in line 46 you have very similar code, try to create and use separate function for this purpose.

**Review by dellachaise — APPROVED** [review]
Great job 👍 

But please add changes according to comments 👇

File: src/averageAges.js (hunk 2 above)
Comment (tiserett) [reply src/averageAges.js]: function is a great idea, done it

**Pull request closed without merging**